ssz/toolz/formatters.py

```diff
@@ -0,0 +1,32 @@
+from eth_utils import (
+    decode_hex,
+    encode_hex,
+)
+
+
+class DefaultFormatter:
```

Comment:
Terminology wise, we might call this a `Codec` and continue using the terms `encode_thing` and `decode_thing`.  It simplifies the naming of things in the dump functions to just be `dump_vector(value, sedes, codec=codec)`